doc/source/drivers/raster/cog.rst

```diff
@@ -91,6 +91,12 @@ General creation options
    0=mathematically lossless, 1.0=visually lossless, usual range [0.5,3].
    Default is 1.0
 
+-  **JXL_ALPHA_DISTANCE=[-1,0,0.1-15]**: (GDAL >= 3.7, libjxl > 0.8.1)
```

Comment:
Can you add some more details about visually lossless and the "usual range"?

Reply:
I've tried to improve based on the information based on the man page of "cjxl" utility (https://www.mankier.com/1/cjxl) that comes with libjxl, and the usage issue of the utility:
```
$ cjxl
[...]
-d maxError, --distance=maxError
    Max. butteraugli distance, lower = higher quality.
    0.0 = mathematically lossless. Default for already-lossy input (JPEG/GIF).
    1.0 = visually lossless. Default for other input.
    Recommended range: 0.5 .. 3.0. Allowed range: 0.0 ... 25.0.
    Mutually exclusive with --quality.
 -a maxError, --alpha_distance=maxError
    Max. butteraugli distance for the alpha channel, lower = higher quality.
    0.0 = mathematically lossless. 1.0 = visually lossless.
    Default is to use the same value as for the color image.
    Recommended range: 0.5 .. 3.0. Allowed range: 0.0 ... 25.0.
```